pkg/testrunner/template/template.go

```diff
@@ -83,3 +84,12 @@ func Render(tmClient kubernetes.Interface, parameters *TestrunParameters, metada
 
 	return testruns, nil
 }
+
+func exposeGardenerVersionToParameters(componentDescriptor componentdescriptor.ComponentList, parameters *TestrunParameters) {
+	for _, component := range componentDescriptor {
+		if component.Name == "github.com/gardener/gardener.version" {
+			parameters.GardenerVersion = component.Version
+			break
```

Comment:
I would rather return instead of breaking out of the loop.
```suggestion
			return
```